client/src/components/Task.js

```diff
@@ -49,13 +49,13 @@ const LabelContainer = styled.div`
   }
 `;
 
-export const Task = ({ done, label, task }) => {
+export const Task = ({ done, label, task, onClick }) => {
   return (
     <>
       <Container>
         <p>{task}</p>
         <LabelContainer done={done}>
-          <button> {done ? "DONE" : "DO"}</button>
+          <button onClick={onClick}> {done ? "DONE" : "DO"} </button>
```

Comment:
It seems like you need to pass that specific prop to that button:

`
<button onClick={onClick} done={done}> {done ? "DONE" : "DO"} </button>

`